api/api-rule-violations-known.list

```diff
@@ -175,6 +175,9 @@ API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,VirtualMachin
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,VirtualMachineStatus,Conditions
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,VirtualMachineStatus,StateChangeRequests
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,VirtualMachineStatus,VolumeSnapshotStatuses
+API rule violation: list_type_missing,kubevirt.io/client-go/apis/flavor/v1alpha1,VirtualMachineClusterFlavorList,Items
+API rule violation: list_type_missing,kubevirt.io/client-go/apis/flavor/v1alpha1,VirtualMachineFlavorList,Items
+API rule violation: list_type_missing,kubevirt.io/client-go/apis/flavor/v1alpha1,VirtualMachineFlavorSpec,CustomProfiles
```

Comment:
This api violation can be fixed. 

Just add `      // +listType=atomic` above the CustomProfiles definition in types.go

Reply:
I didn't know about the `listType` annotation. From the documentation[1] it seems that `+listType=map` would be more appropriate?

[1] - https://github.com/kubernetes/kube-openapi/blob/master/pkg/idl/doc.go#L21

Reply:
yep,  i think map would work